packages/starksheet-webapp/src/provider/EVMProvider.ts

```diff
@@ -190,31 +192,75 @@ export class EVMProvider implements ChainProvider {
   execute = async (calls: Call[], options: { value: number | string })  => {
     const provider = new ethers.providers.Web3Provider(window.ethereum);
     const signer = provider.getSigner();
+    const MULTISEND_CONTRACT_ADDRESS = process.env.REACT_APP_MULTISEND_ADDRESS || "";
 
+    const encodeTransactions = await Promise.all(calls.map(async (call, index) => {
+      let abi = await this.getAbi(call.contractAddress);
+      if(abi.length === 0) {
+        abi = evmWorksheetAbi;
+      }
+      // todo: implement cleanly
```

Comment:
```suggestion
      let abi = await this.getAbi(call.contractAddress);
      // explain why manually setting the ABI here works
      if(abi.length === 0) {
        abi = evmWorksheetAbi;
      }
```

rather than leaving a todo, now that the behavior is implemented, you can add a comment to explain why it is fine to inject a worksheet ABI here, given the usage of the execute function